### ISSUE TITLE
Pico Remote buttons recently stopped working in HA integration lutron_caseta

### ISSUE THREAD

**Issue by glarwill** [gurumitts/pylutron-caseta#95]
I filed [this issue](https://github.com/home-assistant/core/issues/67208) under the lutron_caseta integration in Home Assistant. It was suggested I report the issue here, but add debug log data.

Summary:
I’m using the internal Lutron Caseta integration. In the last 2-3 weeks, my Pico Remote buttons are no longer enabled within the integration. This integration is connected to my Lutron Hub Pro2 (L-BDGPRO2-WH).

When I use "Listen for Events" in the Developer tools page for "lutron_caseta_button_event"s, I see no events firing for Pico Button presses.

I have no issues with Lutron wall switches attached to this hub. The wall switches and the Pico remote buttons show up in the HA integration properly as “devices” (Qty 7), but there are no “entities” associated with the Pico devices only the wall switch devices (Qty 3). One of my Lutron devices is a Ceiling fan controller that has a switch/relay function and buttons to adjust the speed. Only the switch is visible from the integration.

The full debug log is attached below as a text file, but the one Error level message logged is:
```
ERROR (MainThread) [pylutron_caseta.smartbridge] Failed device status subscription: Response(Header=ResponseHeader(StatusCode=ResponseStatus(400, 'BadRequest'), Url='/button/101/status/event', MessageBodyType='ExceptionDetail'), CommuniqueType='ExceptionResponse', Body={'Message': 'This request is not supported'})
```

I created this log file with `$ cat home-assistant.log | grep pylutron > pylutron.log` while logging level was set to DEBUG. Let me know if more/other log data would help.

[pylutron.log](https://github.com/gurumitts/pylutron-caseta/files/8145407/pylutron.log)

**Comment by bdraco** [comment]
https://support.lutron.com/us/en/product/casetawireless/article/bridge-connection/how-to-update-caseta-bridge-or-ra2-select-firmware

**Comment by rborkow** [comment]
I'm might be experiencing the same issue on the `08.07.12f000` firmware. Lutron switches worked fine, and I could see every pico button event coming through my homebridge/pico integration. But my event listener for pico button presses in node-red wasn't seeing any events come through until I restarted the HA container.

Lost my logs after a restart, but I have debug on now and I'll post here if it comes up again.

**Comment by rborkow** [comment]
Confirm it's still occurring on the latest FW and on the new HA release 2022.3.0.

Here's the only error that shows up in the HA logs, debug log attached:
<img width="1275" alt="image" src="https://user-images.githubusercontent.com/2956000/156492755-637a4c4f-e4b4-43dc-8133-d63011b507c9.png">

Log generate as described by @glarwill:
[pylutron.log.gz](https://github.com/gurumitts/pylutron-caseta/files/8174832/pylutron.log.gz)

**Comment by mdonoughe** [comment]
I'm not sure how you got that "was not expecting" message. It looks like maybe your computer or network is having trouble keeping up. pylutron_caseta is taking a long time to initialize and then shortly after initialization it times out checking if the bridge is still there and starts the whole process over. You should not be getting the message "ping was not answered. closing connection."

**Comment by mdonoughe** [comment]
"ping was not answered. closing connection." should be immediately followed by "Reconnecting..." but I see in your log it's fairly consistent in taking 38 seconds.

**Comment by glarwill** [comment]
My issue has vanished with firmware update.

I did try to find a software update function within the App before opening this issue, but found nothing there. I used a different network connection (not behind a pi-hole DNS server) and restarting the hub permitted the software update to start soon after.

I'll check with new container download as well. Finished upgrade to 2022.3.0. Confirmed hub firmware is also at:
'Firmware': {'DisplayName': '08.07.12f000'}

Buttons still work fine here.
